src/stavebarline.js

```diff
@@ -23,21 +23,45 @@ Vex.Flow.Barline = (function() {
   };
 
   Vex.Inherit(Barline, Vex.Flow.StaveModifier, {
-    init: function(type, x) {
+    init: function(type) {
```

Comment:
Looks like this change broke VexTab :-(

`src/barnote.js` creates `BarLine` elements and supplies the `x` parameter to the constructor.